templates/stencil/editor.html.twig

```diff
@@ -106,15 +113,15 @@
                     </div>
                 </div>
                 <div class="col-auto">
-                    <button type="button" id="save-zone-data-{{ rand }}" class="btn btn-primary">
+                    <button type="button" id="save-zone-data-{{ rand }}" class="btn btn-primary" data-bs-toggle="tooltip" data-bs-placement="right" data-bs-title="{{ params['save_zone_data_label']|default(__("Save zone data")) ~ " (Enter)" }}">
```

Comment:
1. `Enter` should be translated.
2. As translation may contains unexpected chars, it is safer to use the `html_attr` escape filter.
```suggestion
                    <button type="button" id="save-zone-data-{{ rand }}" class="btn btn-primary" data-bs-toggle="tooltip" data-bs-placement="right" data-bs-title="{{ (params['save_zone_data_label']|default(__("Save zone data")) ~ ' (' ~ _x('keyboard', 'Enter') ~ ')')|escape('html_attr') }}">
```